picard/file.py

```diff
@@ -370,7 +370,12 @@ def _make_filename(self, filename, metadata, settings=None):
                 if sys.platform == "darwin":
                     new_filename = unicodedata.normalize("NFD", new_filename)
 
-        return os.path.realpath(os.path.join(new_dirname, new_filename))
+        try:
+            return os.path.realpath(os.path.join(new_dirname, new_filename))
+        except FileNotFoundError:
+            # os.path.realpath can fail if cwd doesn't exist
+            return os.path.join(new_dirname, new_filename)
```

Comment:
Not exactly sure about the intended affect here?

Reply:
realpath is using getcwd internally, thus raising a FileNotFoundError. I guess if we just return the non-canonical path in that case, it won't work worse than now, and for sure that works for the examples in the options dialog.

Reply:
```python
    new_path = os.path.join(new_dirname, new_filename)
    try:
        return os.path.realpath(new_path)
    except FileNotFoundError:
        # os.path.realpath can fail if cwd doesn't exist
        return new_path
```

Not sure either, but if it's `realpath` that throws an exception it'll be better to write it as above.

Reply:
Ok, changed the code to that